geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/DeltaWriter.scala

```diff
@@ -213,22 +196,19 @@ class DeltaWriter(
     */
   override def close(): Unit = {
     CloseWithLogging(writer) // also closes `vector`
-    dictionaryWriters.foreach(w => CloseWithLogging(w.writer)) // also closes dictionary vectors
+    CloseWithLogging(dictionaryWriters) // also closes dictionary vectors
     CloseWithLogging(allocator)
   }
 }
 
 object DeltaWriter extends StrictLogging {
 
+  import org.locationtech.geomesa.utils.geotools.RichAttributeDescriptors.RichAttributeDescriptor
+
   import scala.collection.JavaConverters._
 
   // empty provider
-  private val provider = new MapDictionaryProvider()
-
-  private val dictionaryOrdering: Ordering[AnyRef] = new Ordering[AnyRef] {
-    override def compare(x: AnyRef, y: AnyRef): Int =
-      SimpleFeatureOrdering.nullCompare(x.asInstanceOf[Comparable[Any]], y)
-  }
+  private val Provider = new MapDictionaryProvider()
```

Comment:
Any particular reason for the capital P? (No need to change it.)

Reply:
just scala convention for singleton object vals...